src/convertToObject.js

```diff
@@ -13,6 +13,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const makeArraywithoutEmpty = sourceString
+    .split(';').filter(element => element.includes(':'));
```

Comment:
Цей фільтер точно потрібний ?
```suggestion
    .split(';').
```

Reply:
Я думаю, можна і без нього обійтися. Просто в циклі перевіряти, чи там не пустий рядок.